k9mail/src/main/res/values/colors.xml

```diff
@@ -1,7 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <color name="message_list_item_footer_background">#eeeeee</color>
-    
+
+    <color name="light_black">#444</color>
```

Comment:
light black, green and red don't really help knowing where to use it. 

Have you tested how the 'light black' looks on Dark Theme?